docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -105,11 +105,20 @@ protocol being tunneled, a channel ID allowing for additional routing options, a
 Vertiq modules are configured to accept broadcast messages with a protocol value of 0x55, and check the received data buffer for valid :ref:`IQUART data <uart_messaging>`. 
 As such, the DroneCAN tunnel broadcast message allows users to transmit IQUART messages via DroneCAN. Vertiq modules do not support use of the channel ID for additional routing.
 
-A very rudimentary example of using the DroneCAN tunnel message is provided below. The example requires an SLCAN device such as the `Zubax Babel <https://zubax.com/products/adapters/canface>`_, and two :ref:`FTDI devices <connection_guide>`. 
-The hardware is configured as follows:
+A very rudimentary example of using the DroneCAN tunnel message is provided below. The example requires an SLCAN device such as the `Zubax Babel <https://zubax.com/products/adapters/canface>`_ and any 
+software that can emulate COM ports. In this example, we are using `com0com <https://com0com.sourceforge.net/>`_. We will use the virtual COM ports in order to create the following connections:
 
 .. image:: ../_static/manual_images/dronecan/dronecan_tunnel_hardware.png
 
+In order to create the required virtual COM ports using com0com, open com0com, and create a Virtual Port Pair. In this example, we've created COM16 and COM17 which, by default, are connected as depicted in the diagram above.
+
+.. image:: ../_static/manual_images/dronecan/com_0_com.png
+
+On Windows, you can confirm that these ports are reachable through Device Manager.
+
+.. image:: ../_static/manual_images/dronecan/tunnel_device_manager.png
+
+
 In this example, the module has no serial connection to :ref:`IQ Control Center <control_center_start_guide>`. Instead, the Control Center sends messages to a Python script which packages 
```

Comment:
I wonder if this part explaining the general idea of the example should be ahead of the virtual serial port creation details. May be good to say this to get the idea of how it works out there, and then describe how to set it all up